litgpt/model.py

```diff
@@ -304,25 +365,28 @@ def forward(
         # - C*         | attentions's embeddings size
         # - nh_(q,k,v) | number of heads for query, key and value
         # - hs         | head size
-
-        B, T, C = x.size()
+        n_head = self.config.n_head
+        n_query_groups = self.config.n_query_groups
+        rope_n_elem = self.config.rope_n_elem
+        head_size = self.config.head_size
+        B, T, C = x.size()  # batch size, sequence length, embedding dimensionality (n_embd)
```

Comment:
```suggestion
        head_size = self.config.head_size
        n_head = self.config.n_head
        n_query_groups = self.config.n_query_groups
        rope_n_elem = self.config.rope_n_elem
        B, T, C = x.size()  # batch size, sequence length, embedding dimensionality (n_embd)
```

just a small reordering.